### DISCUSSION TITLE
Knowing the Reconstruction neighborhoods in Normandy

### DISCUSSION THREAD

**Discussion by GuillaumeChretien** [quarto-dev/quarto-cli#11705] [Show and Tell]
### Description

Hello, 
I am sharing a project carried out in my public institution to better understand the heritage of housing rebuilt after World War II (for the French speakers !) :
[site web Recoquartiers](https://app-nc.cerema.fr/recoquartiers/)
The site is far from perfect (we had limited time, this was my first project and PDF outputs were initially planned...), but this web version created with Quarto has found some local success and helps raise awareness about actions that need to be initiated. A big thank you to the whole team, and for those interested in Observable JavaScript, take a look at the dashboard with interactive 3D cartography (Maplibre JS linked to a mutable) : 
[https://app-nc.cerema.fr/recoquartiers/carto_interactive.html](https://app-nc.cerema.fr/recoquartiers/carto_interactive.html)

**Comment by mcanouil** [comment]
Thanks for sharing!

Could you edit your post to fix the last link?

**Reply by GuillaumeChretien** [reply]
done, thanks !
